_posts/2015-05-15-Rust-1.0.md

```diff
@@ -0,0 +1,328 @@
+---
+layout: post
+title: "Announcing Rust 1.0"
+author: The Rust Core Team
+---
+
+Today we are very proud to announce the
+[1.0 release of Rust][relnotes], a new programming language aiming to
+make it easier to build reliable, efficient systems. **Rust combines
+low-level control over performance with high-level convenience and
+safety guarantees**. Better yet, it achieves these goals without
+requiring a garbage collector or runtime, making it possible to
+[use Rust libraries as a "drop-in replacement" for C][ffi]. If you'd
+like to experiment with Rust, the
+["Getting Started" section of the Rust book][book] is your best bet.
```

Comment:
So, http://killercup.github.io/trpl-ebook/ is also a thing. It's not official, but it is really nice to read.